bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -185,31 +193,33 @@ Equipment,UPS,PowerSupply,UPS,Uninterruptible Power Supply,
 Equipment,WindGenerator,PowerSupply,Wind Generator,Wind Turbine,
 Equipment,Printer,,Printer,,
 Equipment,Printer3D,Printer,3D Printer,,
-Equipment,Projector,,Projector,"Projectors, Beamer, Beamers",
 Equipment,Pump,,Pump,Pumps,
 Equipment,WaterFeature,Pump,Water Feature,"Waterfall, Pond Pump",
 Equipment,RemoteControl,,Remote Control,Remote Controls,
 Equipment,Sensor,,Sensor,Sensors,
 Equipment,AirQualitySensor,Sensor,Air Quality Sensor,,
 Equipment,CO2Sensor,Sensor,CO2 Sensor,,
 Equipment,COSensor,Sensor,CO Sensor,,
+Equipment,ContactSensor,Sensor,Contact Sensor,"Door Contact, Window Contact",
```

Comment:
I think we should drop the synonyms, so that if someone wants to add custom Door Contact / Window Contact, they won't conflict with the synonyms here.
```suggestion
Equipment,ContactSensor,Sensor,Contact Sensor,,
```

Reply:
Ok I will delete the synonyms. But lets wait for any final inputs apropos your other comment..